src/viewer.js

```diff
@@ -2904,6 +2909,10 @@ function onCanvasClick( event ) {
             );
             this.viewport.applyConstraints();
         }
+        else if( gestureSettings.dblClickToZoom ) {
+            THIS[ this.hash ].lastClickInfo.time = (new Date()).getTime();
```

Comment:
We have a helper for getting the current time, so you should use that: `$.now()`.